### PR TITLE
Adds Ask Us! menu to website main nav

### PR DESCRIPTION
**Pull request by sdellis**
Todo: click behavior and submission form

refs #779

### CODE REVIEW

**Comment by sdellis** [comment]
Screen Shot 2022-04-29 at 3 08 26 PM
Screen Shot 2022-04-29 at 3 08 04 PM
Screen Shot 2022-04-29 at 3 07 51 PM
Screen Shot 2022-04-29 at 3 07 25 PM
Screen Shot 2022-04-29 at 3 07 12 PM

**Comment by sdellis** [comment]
Good catch, @hackartisan! That png is not being used. Just removed it in the last commit.

**Review by hackartisan — APPROVED** [review]

**Comment by tpendragon** [comment]
@sdellis On submit there wasn't any message to users, since the JS wasn't set up to activate the Rails UJS form. I added a commit to do that: https://github.com/pulibrary/pulfalight/pull/1033/commits/ce0e360c55eb51de5b2ddef4de8ad506cb400bdd

**Comment by sdellis** [comment]
Thanks, @tpendragon !

**Comment by sdellis** [comment]
We need to get approval for this from the product owners before this can be merged. @ccleeton @faithc

**Comment by sdellis** [comment]
These are the confirmation messages for each of the feedback forms. We will need updated language for each. Suggest a Correction and Harmful Language submissions go to the same LibAnswers queue but have different subjects so it it will be easy to tell them apart.

Screen Shot 2022-05-03 at 3 07 10 PM
Screen Shot 2022-05-03 at 3 06 29 PM
Screen Shot 2022-05-03 at 3 05 44 PM

**Comment by tpendragon** [comment]
We got confirmation that we can get this in and adjust the layout of it in response to feedback from leadership.

**Comment by ccleeton** [comment]
I added this response text for Ask us to another ticket. Could we change it please?

“Thank you for submitting your question to Special Collections. Someone will reply to you via email soon. We appreciate your patience and look forward to addressing your inquiry.”

**Comment by tpendragon** [comment]
@ccleeton Is it okay if it changes for the response you get from the popup on the individual component suggest a correction too?

**Comment by faithc** [comment]
@tpendragon The automated response for the Ask Us form will affect the response for suggest a correction, or vice versa?

**Comment by tpendragon** [comment]
Right now the message for hitting a components "suggest a correction" button and filling out that form is the same as hitting the "suggest a correction" from the menu bar in the ask us drop-down here, yes.

**Comment by tpendragon** [comment]
@faithc @ccleeton Ah, sorry. Similar question - it's cool for Ask a Question from the menu to have the same response as Ask a Question from a component?

**Comment by faithc** [comment]
Not cool (ha); they should still have different responses.

File: app/views/layouts/application.html.erb
Comment (tpendragon) [inline app/views/layouts/application.html.erb:4]: 👍  Good catch.

**Review by tpendragon — APPROVED** [review]

**Pull request merged**
